src/app/people-isolating/people-isolating.page.ts

```diff
@@ -13,21 +14,34 @@ import { Router } from "@angular/router";
 })
 export class PeopleIsolatingPage implements OnInit {
   isolators$: Observable<Array<Isolator>>;
+  user$: Observable<Volunteer>;
+
   constructor(
     private afs: AngularFirestore,
     public auth: AngularFireAuth,
     private router: Router
-  ) {}
+  ) {
+    this.user$ = this.auth.authState.pipe(
+      switchMap(user => {
```

Comment:
i'm assuming this doesn't work yet, you are connecting to your own Firestore right? (that's pretty cool)

Reply:
So, assuming we have the /users collection up and running with the right county mapping, this should work straight away